docs/index.md

```diff
@@ -90,9 +90,9 @@ Aragon organizations can be upgraded seamlessly using our aragonOS architecture.
 
 The Aragon Network Token, ANT, puts the power into the hands of the people participating in the operation of the Network. Every single aspect of the Network will be governed by those willing to make an effort for a better future.
 
-### Aragon Core
+## Aragon Core
```

Comment:
Why the change on the titles? They are sub-items to `What is Aragon?` as that's the top category of which Aragon Core, Network, Why Aragon and Team are apart of, so I don't see why they should be moved to top level items.

Reply:
@Smokyish there was a kind of odd formatting where some parts are markdown and some are HTML, and the sub-item was H3 (###) but the section header below that was H2 so bigger. I switched them around so that the sub-item would be H2 and the section header below it would be H3. More readable this way. We can make `What is Aragon?` H1 to show it is the top-level item if that makes sense. Or change what I changed from H2 and H3 to H3 and H4, respectively.

Reply:
@john-light `#` is reserved for the title of the page, any titles after that need to be at least `##`. The `<h2>` tags are used, in this case, so they would not appear as items in the ToC, as there's no need for them to appear in the links/menu/navigation. 

This sounds more of a cosmetic change then, which can be done via `stylesheets/extra.css` instead of changing the structure of the page and should probably thus be a PR of it's own.